visualization/app/codeCharta/ui/blacklistPanel/blacklistPanel.component.ts

```diff
@@ -23,6 +25,15 @@ export class BlacklistPanelController implements BlacklistSubscriber {
 		const excluded: BlacklistItem[] = []
 
 		for (const item of blacklist) {
+			const path = item.path
+			item.display_path = path
+
+			if (path.length > this.MAX_PATH_LENGTH) {
+				const shortName = path.slice(-this.MAX_PATH_LENGTH)
+				const replaceBeforePosition = shortName.indexOf("/")
+				item.display_path = `..${shortName.slice(replaceBeforePosition)}`
```

Comment:
Instead of actually cutting of parts of the name, let's use the css functionality instead. It allows to display the dots at the correct spot in all cases: https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow